sale_stock_available_info_popup/models/sale_order.py

```diff
@@ -0,0 +1,40 @@
+# Copyright 2020 Tecnativa - Ernesto Tejeda
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from collections import defaultdict
+
+from odoo import api, fields, models
+
+
+class SaleOrderLine(models.Model):
+    _inherit = "sale.order.line"
+
+    immediately_usable_qty_today = fields.Float(
+        compute="_compute_immediately_usable_qty_today"
+    )
+
+    @api.depends(
+        "product_id",
+        "product_uom_qty",
+        "order_id.commitment_date",
+        "order_id.date_order",
+        "warehouse_id",
+    )
+    def _compute_immediately_usable_qty_today(self):
+        qty_processed_per_product = defaultdict(lambda: 0)
+        self.immediately_usable_qty_today = False
+        for line in self.sorted(key=lambda r: r.sequence):
+            if line.order_id.commitment_date:
+                date = line.order_id.commitment_date
+            else:
+                date = line._expected_date()
+            if not line.display_qty_widget:
+                continue
+            product = line.product_id.with_context(
+                to_date=date, warehouse=line.warehouse_id.id
```

Comment:
Using `line.scheduled_date` should do the same:
```suggestion
            if not line.display_qty_widget:
                continue
            product = line.product_id.with_context(
                to_date=line.scheduled_date, warehouse=line.warehouse_id.id
```